go/service/teams.go

```diff
@@ -236,6 +236,46 @@ func (h *TeamsHandler) TeamAddMember(ctx context.Context, arg keybase1.TeamAddMe
 	return result, nil
 }
 
+func (h *TeamsHandler) TeamAddMembers(ctx context.Context, arg keybase1.TeamAddMembersArg) (err error) {
+	ctx = libkb.WithLogTag(ctx, "TM")
+	debugString := "0"
+	if len(arg.Assertions) > 0 {
+		debugString = fmt.Sprintf("'%v'", arg.Assertions[0])
+		if len(arg.Assertions) > 1 {
+			debugString = fmt.Sprintf("'%v' + %v more", arg.Assertions[0], len(arg.Assertions)-1)
+		}
+	}
+	defer h.G().CTraceTimed(ctx, fmt.Sprintf("TeamAddMembers(%s, %s)", arg.Name, debugString),
+		func() error { return err })()
+	if len(arg.Assertions) == 0 {
+		return fmt.Errorf("attempted to add 0 users to a team")
+	}
+	if err := h.assertLoggedIn(ctx); err != nil {
+		return err
+	}
+	res, err := teams.AddMembers(ctx, h.G().ExternalG(), arg.Name, arg.Assertions, arg.Role)
+	// Check the error after spawning a goroutine to send chat notifications.
+	savedErr := err
```

Comment:
I'm not sure what's going on here, would you mind clearing it up for me?

Reply:
Thanks for pointing this out. This is from when it could partially succeed. Doesn't make sense anymore.